uber/site_sections/registration.py

```diff
@@ -167,6 +167,18 @@ def history(self, session, id):
                                .order_by(Tracking.when).all()
         }
 
+    @ajax
+    def resend_email(self, session, id):
+        message = {''}
+        if id:
+            try:
+                email = session.query(Email).filter(Email.model == 'Attendee', Email.fk_id == id).all()[0]
+                send_email(c.ADMIN_EMAIL, email.dest, email.subject, email.html)
+                message = 'Email Sent!'
+            except ValueError:
```

Comment:
Although I generally like the try/except block format, you may want to have two checks here: one for the email existing, and one for more generic send_email failures. It _should_ be possible to check for a row in the database without throwing an exception. So you'd have something a bit more like (pseudo-code):
```
email = session.blah.first()
if not email:
  return 'No emails found with that ID'
try:
  send_email(blah)
except:
  return 'Could not resend email. Try again or contact your system administrator.'
```